go.mod

```
@@ -6,6 +6,7 @@ require (
 	cuelang.org/go v0.9.2
 	github.com/AlecAivazis/survey/v2 v2.1.1
 	github.com/FogDong/uitable v0.0.5
+	github.com/Masterminds/semver v1.5.0
```

Shouldn't we use `Masterminds/semver/v3` instead, which is already included?

```suggestion
	github.com/Masterminds/semver v1.5.0
```

---

We may want to upgrade this library to the latest version, currently v3.3.1

See: https://github.com/Masterminds/semver/releases

---

We'll create a separate issue to address this.